### PR TITLE
remove reference to newrelic

### PR DESCRIPTION
**Pull request by macav**

### CODE REVIEW

File: ruby_on_rails/README.md [hunk 1]
```diff
@@ -31,7 +31,6 @@ While everyone starts working there are some more things which you should setup.
 They are not optional, but the rest of the team can start working even if those are not in place yet.
 
 1. [GetSentry](getsentry.md)
-1. [NewRelic](newrelic.md)
```
Comment (coorasse) [inline ruby_on_rails/README.md]: why? :)

Comment (macav) [reply ruby_on_rails/README.md]: we don't handle the alerts and don't want to use it, so we agreed with Samuel to deprecate it. for now i disabled only the alerts, but i didn't close the account completely.

Comment (coorasse) [reply ruby_on_rails/README.md]: to me, we need to have a tool to analyze potential performance issues in production. If we don't install newrelic by default anymore I'd like to have an alternative process.

**Comment by macav** [comment]
let's keep it then

**Pull request closed without merging**